libraries/eosiolib/contracts/eosio/map.hpp

```diff
@@ -132,6 +136,15 @@ namespace eosio::kv {
          return internal_use_do_not_use::kv_it_lower_bound(itr, "", 0, _k, _v);
       }
 
+      /**
+       * @ingroup keyvaluemap
+       * 
+       * @brief This struct represents the data type stored in map.
+       * @details You will use the sets of functions and operations associated with this type to
```

Comment:
sets or set?

Reply:
I have no idea... I guess both work?... no?

Reply:
I will change it to set anyway (these are the dev's comments and I did not want to interfere too much with them content unless it was obvious something not clear/good)